src/Commands/CommandFactory.cs

```diff
@@ -95,6 +96,28 @@ private void RegisterCosmosCommands()
         cosmosItem.AddCommand("query", new ItemQueryCommand(GetLogger<ItemQueryCommand>()));
     }
 
+    private void RegisterPostgresCommands()
+    {
+        var pg = new CommandGroup("pg", "PostgreSQL operations - Commands for listing and managing Azure Database for PostgreSQL - Flexible server.");
```

Comment:
var pg = new CommandGroup("postgres", "PostgreSQL operations - Commands for listing and managing Azure Database for PostgreSQL - Flexible server.");